assets/eip-3475/utils/Context.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: CC0
```

Comment:
```suggestion
// SPDX-License-Identifier: CC0-1.0
```

Reply:
this is not required as these files are now deleted